src/GeneralStateTestsFiller/stEWASMTests/utils/convert_wast.py

```diff
@@ -28,7 +28,7 @@ def convert_state_test(file_name):
   for test_case in state_test.keys():
     # conver pre account code
     for addr in state_test[test_case]['pre']:
-      if state_test[test_case]['pre'][addr]['code'] != ['']:
+      if state_test[test_case]['pre'][addr]['code'] != '':
         state_test[test_case]['pre'][addr]['code'] = '0x'+wat2wasm(state_test[test_case]['pre'][addr]['code'], file_name)
 
     if state_test[test_case]['transaction']['data'] != [''] and state_test[test_case]['transaction']['to'] == '':
```

Comment:
Should this also be `''`?

Reply:
The answer is yes.  Pyyaml was returning [''] as the value for empty fields at some point in time.  Not sure what I did to change that...